core/templates/hashfuncs.h

```diff
@@ -74,6 +74,13 @@ static inline uint32_t hash_djb2_one_32(uint32_t p_in, uint32_t p_prev = 5381) {
 	return ((p_prev << 5) + p_prev) + p_in;
 }
 
+/**
+ * Thomas Wang's 64-bit to 32-bit Hash function:
+ * http://web.archive.org/web/20071223173210/http:/www.concentric.net/~Ttwang/tech/inthash.htm
```

Comment:
Both URLs can use HTTPS:
```suggestion
 * https://web.archive.org/web/20071223173210/https:/www.concentric.net/~Ttwang/tech/inthash.htm
```

Reply:
Thank you! Done.